app/cron.yaml

```diff
@@ -30,3 +30,7 @@ cron:
 - description: clean up in test mode
   url: /global/tasks/clean_up_in_test_mode
   schedule: every 60 minutes
+
+# Send email notification when there are many unreviewed records.
+- description: send email notification for review
```

Comment:
Put the sentence you wrote in the previous line to this line instead. It looks clearer.